KILTprotocol/sdk-js
# feat: support account link pallet update

## fixes KILTProtocol/ticket#2048

This adds discovery of whether the account linking pallet has updated to support ethereum alongside substrate addresses.
Extrinsic and query parameter encoding as well as decoding of results will adapt automatically.

## How to test:

As long as https://github.com/KILTprotocol/kilt-node/pull/355 is not yet merged, testing will have to be done against an image built from source on that branch.
Tests for ethereum address linking have therefore been set up to be skipped.
Integration tests against latest & latest-develop images should nevertheless still work.

## Checklist:
